manifests/job.pp

```diff
@@ -20,6 +20,7 @@
   $config,
   $jobname  = $title,
   $enabled  = 1,
+  $sourced  = 0,
```

Comment:
This should be a proper boolean instead of an integer.  However, I believe a `source` and/or `content` parameter in the style of the core `file` resource would be more idiomatic.